src/server/pegasus_server_impl.cpp

```diff
@@ -1560,9 +1560,10 @@ dsn::error_code pegasus_server_impl::start(int argc, char **argv)
 
     ddebug_replica("start to open rocksDB's rdb({})", rdb_path);
 
-    // Here we create a `tmp_data_cf_opts` because we don't want to modify `_data_cf_opts`, which
+    // Here we create a `_tmp_data_cf_opts` because we don't want to modify `_data_cf_opts`, which
     // will be used elsewhere.
-    rocksdb::ColumnFamilyOptions tmp_data_cf_opts = _data_cf_opts;
+    _tmp_data_cf_opts = _data_cf_opts;
```

Comment:
'tmp' is not good name for a member variable.
hosw about rename '_data_cf_opts' to '_base_data_cf_opts', and rename '_tmp_data_cf_opts' to '_table_data_cf_opts' ?

Reply:
`_data_cf_opts` involves too many files, not necessary.
i will rename `_tmp_data_cf_opts ` to `_table_data_cf_opts `